loadtest/store/memstore/store.go

```diff
@@ -50,18 +70,69 @@ func (s *MemStore) Clear() {
 	s.config = nil
 	s.emojis = []*model.Emoji{}
 	s.posts = map[string]*model.Post{}
+	s.postsQueue.Reset()
 	s.teams = map[string]*model.Team{}
 	s.channels = map[string]*model.Channel{}
 	s.channelMembers = map[string]map[string]*model.ChannelMember{}
+	s.channelMembersQueue.Reset()
 	s.teamMembers = map[string]map[string]*model.TeamMember{}
 	s.users = map[string]*model.User{}
+	s.usersQueue.Reset()
 	s.statuses = map[string]*model.Status{}
+	s.statusesQueue.Reset()
 	s.reactions = map[string][]*model.Reaction{}
 	s.roles = map[string]*model.Role{}
 	s.license = map[string]string{}
 	s.channelViews = map[string]int64{}
 }
 
+func (s *MemStore) setupQueues(config *Config) error {
+	setups := []struct {
```

Comment:
I was thinking if there was a better way to do this. And I'm sure you have already thought about it. Right now, if one needs to add a new queue, they would need to declare a queue field, a new config max field, and add the setup struct here. Wondering what alternative methods you thought about and decided on this one ?

Reply:
I've thought about it for a while. In the end it's not like I've saved much repetition but at least we call the queue factory function and do the error checking in one place only, which was my goal.

The easiest approach would be to simply repeat everything for the 4 queues.  
I can revert to that one if the abstraction is confusing. Just didn't think to invest too much effort on an internal setup function.